packages/gatsby/src/redux/reducers/jobs.js

```diff
@@ -10,31 +10,27 @@ module.exports = (state = { active: [], done: [] }, action) => {
         throw new Error(`An ID must be provided when creating or setting job`)
       }
       const index = _.findIndex(state.active, j => j.id === action.payload.id)
-      const mergedJob = _.merge(state.active[index], {
```

Comment:
Was there a reason merge was happening outside of the branch that was using it? _.merge appears to be immutable from the docs so this appears to be an entirely useless call on initial creation.

Reply:
Probably no reason. Moving it inside conditional makes more sense